main.py

```diff
@@ -104,7 +105,13 @@
             file.write(filedata)
 
     # Language folder conversion
-    LANGUAGE_FOLDER = os.path.join(TARGET_FOLDER, "resources", "language")
+    data = etree.parse(fileToSearch).getroot()
```

Comment:
Might be easier to just check if `TARGET_FOLDER` has `skin.` in the last part of the path?